### DISCUSSION TITLE
I can hear audio, but there's no window?

### DISCUSSION THREAD

**Discussion by puyoxyz** [ppy/osu#15625] [Q&A] [answered]
Running Artix Linux, GNOME on Wayland. I can hear the game, but it doesn't open (there's no osu window). I'm using the AppImage.

- network.log https://paste.puyo.xyz/aveyasaxiw
- performance.log https://paste.puyo.xyz/eqesorujuv
- runtime.log https://paste.puyo.xyz/eyefeyuram

**Comment by gagahpangeran** [comment, marked as answer]
This is probably some bug or weird case of SDL on wayland.

Possible solution that I saw from report in this repo is running the game with env `SDL_VIDEODRIVER=wayland` and see if it's running on wayland.

If it's still not running, you could try running on X session. If it's running on X session, then my only advice is just playing on X session.

Currently we dont have resource to investigate any bug on wayland and report it to SDL.

**Reply by puyoxyz** [reply]
Setting SDL_VIDEODRIVER=wayland changed nothing, but setting SDL_VIDEODRIVER=x11 made it run on XWayland and it worked

**Comment by protheory8** [comment]
Starting osu! in X11 mode (by setting `SDL_VIDEODRIVER` to `x11`), changing the window from fullscreen to windowed in settings and then closing osu! and launching it in Wayland mode (by setting `SDL_VIDEODRIVER` to `wayland`) fixes this for me.

**Comment by peppy** [comment]
https://github.com/libsdl-org/SDL/issues/5416 maybe related.

**Reply by puyoxyz** [reply]
doubt it, since the issue in this discussion turned out to be just with fullscreen, and the issue you linked doesn't say it only happens in fullscreen
